src/pages/explore.tsx

```diff
@@ -1,554 +1,520 @@
+      timeField: '_timesinceepoch',
+    },
+  ],
+});
 
-  const theme = useTheme();
-  if (!hasInitialQueryParams(query)) {
-    syncStateToUrl();
+logsQueryRunner.subscribeToEvent(SceneObjectStateChangedEvent, event => {
+  if (typeof event.payload.newState !== 'undefined') {
+    if (event.payload.newState['data'].state === 'Done') {
+      queryComponent.setLogsLoading(false);
+    } else if (event.payload.newState['data'].state === 'Loading') {
+      queryComponent.setLogsLoading(true);
+    } else if (event.payload.newState['data'].state === 'Error') {
+      logsNodeStats.setCount(-1, -1);
+    }
   }
+});
 
-  const loadData = (callback: Dispatch<SetStateAction<any>>) => {
-    getDataSourceSrv()
-      .get(intState.dataSource)
-      .then(function(datasourceApi) {
-        const interval = rangeUtil.calculateInterval(intState.timeRange, 30).interval;
-        datasourceApi
-          .query({
-            range: intState.timeRange,
-            scopedVars: {
-              __interval: { text: interval, value: interval },
-            },
-            targets: [
-              {
-                // @ts-ignore
-                query: query.queryString,
-                bucketAggs: [
-                  {
-                    type: 'date_histogram',
-                    id: '1',
-                    settings: { interval: interval },
-                  },
-                ],
-              },
-              {
-                // @ts-ignore
-                format: 'logs',
-                isLogsQuery: true,
-                metrics: [
-                  {
-                    type: 'logs',
-                  },
-                ],
-                query: query.queryString,
-                bucketAggs: null,
-              },
-            ],
-          })
-          // @ts-ignore
-          .toPromise()
-          .then(function(response) {
-            const metricsResponse = response.data[0];
-            const dataQueryResponseData = response.data[1];
-
-            const metadata = {
-              metrics: {
-                shards: metricsResponse.meta.shards,
-              },
-              logs: {
-                shards: dataQueryResponseData.meta.shards,
-              },
+/**
+ * This custom transform operation is used to rewrite the _source field to an ansi log line, as
+ * well as initialize the meta information used for debugging purposes.
+ */
+const logsResultTransformation: CustomTransformOperator = () => (source: Observable<DataFrame[]>) => {
+  return source.pipe(
+    map((data: DataFrame[]) => {
+      if (data.length > 0 && data[0].meta['shards']) {
+        logsNodeStats.setCount(data[0].meta['shards'].total, data[0].meta['shards'].failed);
+      }
+      return data.map((frame: DataFrame) => {
+        return {
+          ...frame,
+          fields: frame.fields.map(field => {
+            // todo - this should use the config value "message field name"
+            if (field.name === '_source') {
```

Comment:
This will continue to return all the fields due to the statement immediately after this if. This code is only replacing the _source field for rendering purposes to an ansi log line.